config/application.rb

```diff
@@ -11,6 +11,13 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 7.0
 
+    config.middleware.insert_before 0, Rack::Cors do
+      allow do
+        origins 'http://localhost:3000'
+        resource '*', headers: :any, methods: [:get, :post, :put, :patch, :delete, :options, :head]
+      end
+    end   
+
```

Comment:
The code seems appropriate for allowing cross-origin requests from http://localhost:3000 in a development or testing environment.